### DISCUSSION TITLE
Can we search for msrp?

### DISCUSSION THREAD

**Discussion by gpuhunting** [jef/streetmerchant#1036] [Help]
### Description

Is there any way to limit the searches of items to msrp? Scalpers have moved to listing their goods on Amazon and newegg knowing we're using bots to search for items and frantically clicking buy when something hits.

**Comment by jef** [comment]
You can set `MAX_PRICE_*` in your `.env` to circumvent this. Take a look at the https://github.com/jef/streetmerchant#customization and see if you can find what you're looking for!

Good luck!
